providers/src/airflow/providers/apache/kafka/hooks/consume.py

```diff
@@ -23,6 +23,18 @@
 from airflow.providers.apache.kafka.hooks.base import KafkaBaseHook
 
 
+class KafkaAuthenticationError(Exception):
+    """Custom exception for Kafka authentication failures."""
+
+    pass
+
+
+def error_callback(err):
+    """Handle kafka errors."""
+    print("Exception received: ", err)
```

Comment:
should we try to use logger instead of a print here?

Reply:
i think in other operators they use logger instead

Reply:
Do we need this print statement here? Perhaps we can remove it.